packages/es-components/src/components/base/icons/Icon.js

```diff
@@ -2,48 +2,36 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
-import { regular } from './icon-definitions';
-
 const StyledIcon = styled.i`
   display: inline-block;
-  font-family: bds-func-icons !important;
-  font-size: ${props => props.fontSize};
-  font-style: normal;
-  font-weight: inherit;
-  font-variant: normal;
-  line-height: 1;
-  speak: none;
+  font-size: ${props => props.size};
```

Comment:
Changing this prop name is technically a breaking change. Do we want to make a new major version for this update?

Reply:
Oh I see! I got a little mixed up there.